src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx

```diff
@@ -74,11 +75,32 @@ export default class Summary extends React.Component {
     });
   }
 
-  showApplicationSummary() {
+  showExitDiagnostics() {
```

Comment:
>showExitDiagnostics [](start = 2, length = 19)

Should include all info:

appExitCode: -7300
appExitSpec:
  phrase: APP_SUBMISSION_YARN_EXCEPTION
  issuer: PAI_LAUNCHER
  causer: USER_SUBMISSION
  type: USER_FAILURE
  stage: SUBMITTING
  behavior: NON_TRANSIENT
  reaction: NEVER_RETRY
  reason: "Failed to submit App to YARN RM due to YarnException, maybe App is non-compliant"
  repro:
    - "Submit a job to invalid virtual cluster"
  solution:
    - "Check diagnostics and revise your job config"

appExitMessages: {
  container: "Yarn strart_container.sh syntax error XXX "
  runtime:
    reason:
    solution:
    trigger:
    originalUserExitCode:
    caughtSignal:
    caughtException:
    matchedUserLogString:
    dockerStatus:
    errorLogs:
      user:
      runtime:
      disk:
  launcher: "Submit to unknown queue: XXX"
},

appExitTriggerMessage: "FailedTaskCount 3 has reached MinFailedTaskCount 3 in TaskRole PS_WORKER"
appExitTriggerTaskRoleName: PS_WORKER
appExitTriggerTaskRoleIndex: 2